.npmrc

```diff
@@ -0,0 +1 @@
+engine-strict=true
```

Comment:
Are we sure about this? If this is set to true the package instalattion will not fail (it will not consider installing it) and then in the project building we are going te realize this package was not succedded installed.. What do you think?

Reply:
No, what this does is it prevents you to install the dependencies (`npm install fails`) and it shows that error so the dev can update the node and npm versions. I think it's ok to force people to have a minimum node and npm version in order to run the bootstrap so the project starts with the best setup possible

Reply:
Yeah my bad.
```
If set to true, then npm will stubbornly refuse to install (or even consider installing) any package that claims to not be compatible with the current Node.js version.
```